src/auro-drawer-content.js

```diff
@@ -44,6 +46,17 @@ export class AuroDrawerContent extends LitElement {
 
   static get properties() {
     return {
+
+      /**
+       * Defines whether the component should be light colored for use on dark backgrounds.
```

Comment:
This description does not actively reflect what's going on. From what I see, this attribute is being applied to close button.